docs/source/llm_quantization/usage_guides/quantization.mdx

```diff
@@ -86,10 +86,26 @@ from accelerate import init_empty_weights
 with init_empty_weights():
     empty_model = AutoModelForCausalLM.from_pretrained(model_name, torch_dtype=torch.float16)
 empty_model.tie_weights()
-quantized_model = load_quantized_model(empty_model, save_folder=save_folder, device_map="auto", disable_exllama=False)
+quantized_model = load_quantized_model(empty_model, save_folder=save_folder, device_map="auto", disable_exllama=False, disable_exllamav2=True)
 ```
 
-Note that only 4-bit models are supported with exllama kernels for now. Furthermore, it is recommended to disable the exllama kernel when you are finetuning your model with peft.
+With the release of the exllamav2 kernel, you can get faster inference speed compared to the exllama kernels. You just need to 
```

Comment:
The default is already `disable_exllamav2=False` so I am not sure to understand. Are these new kernels selected by default?

Reply:
I clarified the doc, thanks.

Reply:
Yeah, those are selected by default. I will reformulate the doc so that it is clearer.